### ISSUE TITLE
os.execute's PoolStringArray type parameter is bugged 

### ISSUE THREAD

**Issue by ghost** [godotengine/godot#22412]
**Godot version:**
3.1
**OS/device including version:**
windows 10, 64-bit
**Issue description:**
Using the example from the docs 
`OS.execute('ping', ['-c', '3', 'godotengine.org'])`

you will receive:
![image](https://user-images.githubusercontent.com/36117592/45971403-cbabfa00-bfed-11e8-9f60-21bd5d71a259.png)

Then, you can fix that by doing:
```
var output = []
OS.execute('ping', ['-c', '3', 'godotengine.org'], true, output) 
print(output)
```

However, you will now get:
```
Parser Error: At 'execute()' call, argument 2. Assigned type (Array) doesn't match the function argument's type (PoolStringArray).
```

**Comment by LikeLakers2** [comment]
Can reproduce in latest build from https://hugo.pro/projects/godot-builds/. Windows 10, 64-bit. Godot does not cast an array of strings to a PoolStringArray automatically.

However, this:
```
[Bad value for option -c.
]
```
is not an error. @girng the `-c` option is for specifying a "Routing compartment identifier" (whatever that is), according to `ping -?`. If you're looking to specify how many pings to send, use `-n`.

**Comment by girng** [comment]
yeah, looks like the -c is from https://linux.die.net/man/8/ping, not the windows version. i'll edit that out

**Comment by coppolaemilio** [comment]
Encountered the same issue as well.

**Comment by akien-mga** [comment]
Fixed the documentation issue in 9108b5fba, but the `Parser Error: At 'execute()' call, argument 2. Assigned type (Array) doesn't match the function argument's type (PoolStringArray).` is still valid and likely related to GDScript typing.
